namer/marathon/src/main/scala/io/buoyant/namer/marathon/AppIdNamer.scala

```diff
@@ -1,9 +1,15 @@
-package io.buoyant.marathon.v2
+package io.buoyant.namer.marathon
```

Comment:
I needed to move this because I was running into an issue where adding the `admin` module as a dependency to the `marathon` module was causing some sort of circular dependency issue at build time. It returned a NPE when trying to build `marathon`.